adsp.freesurround/resources/language/English/strings.po

```diff
@@ -22,15 +22,15 @@ msgid "Free Surround"
 msgstr ""
 
 msgctxt "#30001"
-msgid "Free Surround ..."
+msgid "Free Surround..."
 msgstr ""
 
 msgctxt "#30002"
-msgid "FreeSurround, a new surround processor. Is based on foo_dsp_fsurround from pro_optimizer on foobar2000 0.9.x."
+msgid "FreeSurround, is a new surround processor. Is based on foo_dsp_fsurround found on pro_optimizer in foobar2000 0.9.x."
 msgstr ""
 
 msgctxt "#30003"
-msgid "FreeSurround is meant to be the KODI equivalent of your hi-fi receiver's Dolby ProLogic II button (except that it uses a different algorithm).\n\nIts purpose is to decode surround information from your stereo music (including mp3, of course), and to produce multichannel output from it. It works with any stereo music containing surround info in a coding scheme compatible with any the following decoders:\n - Dolby Surround\n - Dolby ProLogic\n - Dolby ProLogic II\n - SRS CircleSurround\n - SRS CircleSurround II\n - DTS Neo:6\n - Lexicon Logic7\n - Stereo Quadrophonic (SQ)\n - Quadrophonic Stereo (QS)\n\nIs based on foo_dsp_fsurround from pro_optimizer on foobar2000 0.9.x"
+msgid "FreeSurround is meant to be the Kodi equivalent of your hi-fi receiver's Dolby ProLogic II button (except that it uses a different algorithm).\n\nIts purpose is to decode surround information from your stereo music including mp3, and to produce multichannel output from it. It works with any stereo music containing surround info in a coding scheme compatible with any the following decoders:\n - Dolby Surround\n - Dolby ProLogic\n - Dolby ProLogic II\n - SRS CircleSurround\n - SRS CircleSurround II\n - DTS Neo:6\n - Lexicon Logic7\n - Stereo Quadrophonic (SQ)\n - Quadrophonic Stereo (QS)\n\nIs based on foo_dsp_fsurround from pro_optimizer in foobar2000 0.9.x"
```

Comment:
No idea this crashes kodi for me so I cant tell what's needed only corrected the obvious.